src/common/config/metadata/value.go

```diff
@@ -0,0 +1,143 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metadata
+
+import (
+	"errors"
+
+	"github.com/goharbor/harbor/src/common/utils/log"
+)
+
+var (
+	// ErrNotDefined ...
+	ErrNotDefined = errors.New("configure item is not defined in metadata")
+	// ErrTypeNotMatch ...
+	ErrTypeNotMatch = errors.New("the required value doesn't matched with metadata defined")
+	// ErrInvalidData ...
+	ErrInvalidData = errors.New("the data provided is invalid")
+	// ErrValueNotSet ...
+	ErrValueNotSet = errors.New("the configure value is not set")
+)
+
+// ConfigureValue - struct to hold a actual value, also include the key of config metadata.
+type ConfigureValue struct {
+	Key   string `json:"key,omitempty"`
```

Comment:
the key is actually the name of Item? It's confused by the definition.

Reply:
Rename key to name